web/modules/custom/joinup_workflow/joinup_workflow.module

```diff
@@ -246,3 +266,26 @@ function joinup_workflow_is_motivation_required($entity_type_id, $workflow_id, $
 
   return !empty($required_transitions[$entity_type_id][$workflow_id]) && in_array($transition_id, $required_transitions[$entity_type_id][$workflow_id]);
 }
+
+/**
+ * Change notification when saving content as draft.
+ *
+ * @param array $form
+ *   The form array.
+ * @param \Drupal\Core\Form\FormStateInterface $form_state
+ *   The form state object.
+ */
+function joinup_workflow_message_form_submit_alter(array &$form, FormStateInterface $form_state) {
+  $node = $form_state->getFormObject()->getEntity();
+  if ($node->get('field_state')->getString() == 'draft') {
+    \Drupal::messenger()->deleteByType('status');
+    $args = [
+      '@type' => node_get_type_label($node),
+      '%title' => $node->getTitle(),
```

Comment:
Instead of use $node->getTitle() i use $node->toLink()->toString() because generates the HTML link to an entity and i can remove the argument ':url'.

Reply:
@saidatom initially I was reluctant to your solution because tokens such as `%title` used to be sanitized and HTML markup ends by being escaped. However, after testing manually I was surprised to find that it just works. I've tracked down just to get an explanation, and, yes, the fact the secret is that `$node->toLink()->toString()` doesn't actually return a string, but a `GeneratedLink` instance and that implements `MarkupInterface`. That means in this line https://git.drupalcode.org/project/drupal/-/blob/9.2.x/core/lib/Drupal/Component/Render/FormattableMarkup.php#L260 the HTML is not escaped but preserved. Nice!